src/Core/src/Handlers/Layout/LayoutHandlerUpdateArgs.cs

```diff
@@ -0,0 +1,20 @@
+﻿#nullable enable
+
+namespace Microsoft.Maui.Handlers
+{
+	/// <summary>
+	/// Communicates information from an ILayout about updates to an ILayoutHandler
+	/// </summary>
+	public struct LayoutHandlerUpdateArgs
```

Comment:
Should we use something like this for the scroll request args as well? @PureWeen I think you had something that also needed a type.

We probably need to be consistent.

Should we have a `record` for this?

Reply:
Yes, this should be a record.

Reply:
Okay, changed to a record.